kratos/tests/test_tikz_output_process.py

```diff
@@ -12,10 +12,12 @@ def GetFilePath(fileName):
     return os.path.join(os.path.dirname(os.path.realpath(__file__)), fileName)
 
 class TestTikZOutputProcess(KratosUnittest.TestCase):
-    def test_ascii_tikz_output_2D(self):
+    @classmethod
```

Comment:
are you sure this still works as intended?
I thought this is python-specific syntax for the unittests?

Reply:
Was the only way to make Codacy stop complaining, and does it work

Reply:
I am not sure if codacy knows everything...
If it still works then its ok I guess